autoconfigure/pom.xml

```diff
@@ -72,8 +72,15 @@
         <configuration>
           <!-- https://github.com/spring-projects/spring-boot/issues/3426 transitive exclude doesn't work -->
           <excludeGroupIds>
-            io.zipkin.zipkin2,io.zipkin.reporter2,org.springframework.boot,org.springframework,com.fasterxml.jackson.core,com.google.auto.value,com.google.guava,com.squareup.moshi,com.squareup.okio,com.squareup.okhttp
+            com.github.openzipkin.zipkin,io.zipkin.reporter2,org.springframework.boot,org.springframework,com.fasterxml.jackson.core,com.google.auto.value,com.google.guava,org.reactivestreams,com.google.code.findbugs,javax.annotation,org.slf4j,io.netty,io.micrometer,org.hdrhistogram,org.latencyutils
```

Comment:
this first entry needs to change back to `io.zipkin.zipkin2` before merge. basically a global replace back needs ot happen